### DISCUSSION TITLE
No settings option

### DISCUSSION THREAD

**Discussion by adam-paz** [Obsidian-TTRPG-Community/fantasy-statblocks#370] [Q&A]
Anyone have any idea why i dont have the option to view/edit settings? 
<img width="621" alt="image" src="https://github.com/javalent/fantasy-statblocks/assets/22757958/ab2bf997-6a58-44ad-9171-cc9250165ea3">

<img width="725" alt="image" src="https://github.com/javalent/fantasy-statblocks/assets/22757958/e3f43498-2334-48c2-b35f-2a3fbd7a32b2">

**Comment by hibeatles** [comment]
After I modified the custom layout, the setting icon disappeared after restarting Obsidian.
